src/client/datascience/interactive-ipynb/nativeEditor.ts

```diff
@@ -602,7 +602,7 @@ export class NativeEditor extends InteractiveBase implements INotebookEditor {
         if (this.previouslyNotTrusted && this.model?.isTrusted) {
             this.previouslyNotTrusted = false;
             // Tell UI to update main state
-            this.postMessage(InteractiveWindowMessages.TrustNotebookComplete).ignoreErrors();
+            await this.postMessage(InteractiveWindowMessages.TrustNotebookComplete);
```

Comment:
Why do we need to `await`?

Reply:
```suggestion
            this.postMessage(InteractiveWindowMessages.TrustNotebookComplete).ignoreErrors();
```

Sorry, left this in by accident while debugging